### ISSUE TITLE
`wp option list` should limit output to some reasonable length

### ISSUE THREAD

**Issue by borekb** [wp-cli/wp-cli#3447]
Just encountered this: I've installed a fresh WP site, loaded it in browser and then tried to run `wp option list` (WP-CLI 0.24.1). The output was huge, basically blocked by console and I had to kill the command.

The root cause was that the "WordPress News" metabox on admin dashboard fetches a RSS feeed that was 60kB long and stored it to a transient like `_transient_feed_d117b5738fbd35bd8c0391cda1f2b5d9`. This made _every line_ of the `option list` output 60,000 characters long and the full output was something like 60 MB! Somehow I do not remember having this problem before but maybe it's because I don't use `wp option list` often.

It made me think this:

- Maybe transients should not be part of the output by default. When I want to list options, I want to see them but not necessarily transients. I'll probably open another issue to discuss this. _UPDATE: done https://github.com/wp-cli/wp-cli/issues/3448._
- The output should perhaps be limited to something reasonable, like, 200 characters per field or something. Most options will fit easily and for the long ones, one could always use the `wp option get` to see more.

**Comment by danielbachhuber** [comment]
Introducing a new `--no-transients` flag seems sufficient to me.

**Comment by borekb** [comment]
I still think `wp option list` should not be outputting 16,000 character long lines by default but I'll be able to work around it with the `--no-transients` flag.
